File: crates/atlas-cli/tests/fixtures/php/class.php
<?php

interface GreeterInterface {
    public function greet();
}

class Greeter implements GreeterInterface {
    public $name;

    public function greet() {
        return "Hello, " . $this->name;
    }
}

$g = new Greeter();
$g->name = "World";
echo $g->greet();

File: crates/atlas-cli/tests/fixtures/csharp/class.cs
interface IGreeter {
    string Greet();
}

class Greeter : IGreeter {
    public string Name { get; set; }

    public string Greet() {
        return "Hello, " + Name;
    }

    static void Main() {
        var g = new Greeter { Name = "World" };
        g.Greet();
    }
}
